Missing `fast-fifo` module after last `electron-builder` upgrade

All CI builds from `main` are failing (at least on macOS for aarch64) since #9539 got merged:

```
Uncaught Exception:
Error: Cannot find module 'fast-fifo'
Require stack:
- /Applications/Rancher Desktop.app/Contents/Resources/app.asar/node_modules/streamx/index.js
- /Applications/Rancher Desktop.app/Contents/Resources/app.asar/node_modules/tar-stream/extract.js
- /Applications/Rancher Desktop.app/Contents/Resources/app.asar/node_modules/tar-stream/index.js
at Module._resolveFilename (node:internal/modules/cjs/loader:1390:15)
at s._resolveFilename (node:electron/js2c/browser_init:2:130470)
at defaultResolveImpl (node:internal/modules/cjs/loader:1032:19)
at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1037:22)
at Module._load (node:internal/modules/cjs/loader:1199:37)
at c._load (node:electron/js2c/node_init:2:17993)
at TracingChannel.traceSync (node:diagnostics_channel:328:14)
at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)
at Module.require (node:internal/modules/cjs/loader:1470:12)
at require (node:internal/modules/helpers:147:16)
``

--- a/package.json
+++ b/package.json
@@ -134,7 +134,7 @@
     "css-loader": "7.1.2",
     "ejs": "3.1.10",
     "electron": "39.2.3",
-    "electron-builder": "26.3.0",
+    "electron-builder": "26.2.0",
     "eslint": "9.39.1",
     "eslint-plugin-vue": "10.5.1",
     "extract-zip": "2.0.1",

